config/locales/es.yml

```diff
@@ -772,38 +771,38 @@ es:
           filer_provided_over_half_housing_support: "¿El contribuyente o contribuyentes aportaron más de la mitad de los gastos de vivienda de esta persona?"
           filer_provided_over_half_support: "¿El contribuyente o contribuyentes aportaron más del 50% de la manutención de esta persona?"
           first_name: Primer Nombre
-          got_married_during_tax_year: A. En caso afirmativo, ¿se casó en 2024?
+          got_married_during_tax_year: A. If Yes, Did you get married in 2024?
           had_disability: Discapacidad total y permanente
-          issued_an_identity_protection_pin: Emitió un PIN de protección de identidad (IPPIN)
-          issued_ippin: Se le emitió un IPPIN
+          issued_an_identity_protection_pin: Issued an identity protection PIN (IPPIN)
+          issued_ippin: Issued IPPIN
```

Comment:
just passing through here but why is everything changing to English here? also not sure who's owning this PR getting merged

Reply:
this section is gyr/13614-c stuff. it doesn't have real translations yet.